.github/workflows/autoRelease.yaml

```diff
@@ -0,0 +1,20 @@
+# When new tags gets created a new release would be automatically created
+name: Auto Release on new Tag
```

Comment:
Do we need to use this same workflow for airgun and nailgun?

Reply:
OOPs , that change shouldnt be part of this PR ! Removing ....